easybuild/easyconfigs/c/CellTypist/CellTypist-1.6.2-foss-2023a.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'CellTypist'
+version = '1.6.2'
+
+homepage = 'https://www.celltypist.org/'
+description = "A tool for semi-automatic cell type annotation"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('scikit-learn', '1.3.1'),
+    ('openpyxl', '3.1.2'),
+    ('scanpy', '1.9.8'),
+    ('leidenalg', '0.10.2'),
+]
+
+exts_list = [
+    ('requests', '2.31.0', {
+        'checksums': ['942c5a758f98d790eaed1a29cb6eefc7ffb0d1cf7af05c3d2791656dbd6ad1e1'],
+    }),
+    ('click', '8.1.7', {
+        'checksums': ['ca9853ad459e787e2192211578cc907e7594e294c7ccc834310722b41b9ca6de'],
+    }),
```

Comment:
@pavelToman Please include as dependency along with `Python-bundle-PyPI`, this bundle provides both `requests` and `click` so there's no need to include them here directly as extensions.

You can still stick to using `PythonBundle` even though there's only one extension (CellTypist itself)